managed/services/telemetry/config.default.yml

```diff
@@ -24,45 +24,45 @@ telemetry:
       - metric_name: "pmm_server_alert_rules_number"
         value: 1
 
-  - id: PMMServerSTTEnabled
+  - id: PMMServerAdvisorsEnabled
     source: PMMDB_SELECT
-    query: (CASE WHEN stt->'stt_disabled' = 'false' THEN '1' ELSE '0' END) AS stt_enabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS stt
-    summary: "PMM Server Security Thread Tool feature enabled/disabled"
+    query: (CASE WHEN advisors->'enabled' = 'true' THEN '1' WHEN advisors->'enabled' = 'null' THEN '1' ELSE '0' END) AS advisors_disabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS advisors
+    summary: "PMM Advisors Enabled"
     data:
-      - metric_name: "pmm_server_stt_enabled"
-        column: "stt_enabled"
-
-  - id: PMMServerAdvisorsDisabled
-    source: PMMDB_SELECT
-    query: (CASE WHEN advisors->'enabled' = 'true' THEN '0' ELSE '1' END) AS advisors_disabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS advisors
-    summary: "PMM Advisors Disabled"
-    data:
-      - metric_name: "pmm_server_advisors_disabled"
-        column: "advisors_disabled"
+      - metric_name: "pmm_server_advisors_enabled"
```

Comment:
do we really need to rename metric name? Was it discussed with @rnovikovP?

Reply:
Roma was Ok'ish with that :)